Riot/Assets/de.lproj/Vector.strings

```diff
@@ -1622,6 +1622,8 @@
 // MARK: Threads
 "room_thread_title" = "Thread";
 "room_accessibility_thread_more" = "Mehr";
+room_accessibility_record_voice_message = "Sprachnachricht aufnehmen";
+room_accessibility_record_voice_message_hint = "Doppeltippen und halten um aufzunehmen.";
```

Comment:
For our translated strings, we manage them all though translate.element.io as otherwise there can be merge conflicts when updating others. Would you be able to revert this commit and then the next time we sync with Weblate you'll be able to add them there instead.

Reply:
@pixlwave sorry, looks like I got ahead of myself there :-).
Reverted with commit 612795d05273d79ad92ddd5b3ba0022b34d441ea.